sdk/eventhubs/azure_messaging_eventhubs/src/common/recoverable/claims_based_security.rs

```diff
@@ -60,31 +59,9 @@ impl RecoverableClaimsBasedSecurity {
         .await
     }
 
-    fn should_retry_claims_based_security_response(e: &azure_core::Error) -> ErrorRecoveryAction {
-        match e.kind() {
-            AzureErrorKind::Amqp => {
-                warn!(err=?e, "Amqp operation failed: {:?}", e.source());
-                if let Some(e) = e.source() {
-                    debug!(err=?e, "Error: {e}");
-
-                    if let Some(amqp_error) = e.downcast_ref::<Box<AmqpError>>() {
-                        RecoverableConnection::should_retry_amqp_error(amqp_error)
-                    } else if let Some(amqp_error) = e.downcast_ref::<AmqpError>() {
-                        RecoverableConnection::should_retry_amqp_error(amqp_error)
-                    } else {
-                        debug!(err=?e, "Non AMQP error: {e}");
-                        ErrorRecoveryAction::ReturnError
-                    }
-                } else {
-                    debug!("No source error found");
-                    ErrorRecoveryAction::ReturnError
-                }
-            }
-            _ => {
-                debug!(err=?e, "Non AMQP error: {e}");
-                ErrorRecoveryAction::ReturnError
-            }
-        }
+    fn should_retry_claims_based_security_response(e: &AmqpError) -> ErrorRecoveryAction {
+        warn!("Amqp operation failed: {:?}", e);
```

Comment:
I see we capitalize AMQP in error messages most of the time
```suggestion
        warn!("AMQP operation failed: {:?}", e);
```